source/routes/delete/month.js

```diff
@@ -0,0 +1,37 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/* *
+Delete month page json object and all bullet documents in that month page.
+*/
+module.exports = {
+    '/delete/month': {
+        methods: ['delete'],
+        fn: function(req, res, next) {
+            //get month page by id
+            db.get(req.body._id)
+            .then((response) => {
+                //access all bullet documents' id
+                response.bullets.forEach((bullet, index) => {
+                    db.get(bullet)
+                    .then(function(doc){
+                        //delete bullet document
+                        return db.remove(doc._id, doc._rev);
+                    })
+                    .catch((err) => {
+                        console.log(err);
+                        res.send("error caused by cannot get bullet Json by id");
+                    });
+                })
+                //delete month page
+                db.remove(response._id, response._rev);
+                res.send("delete success");
```

Comment:
I believe this should also only return "success" - you know when you have sent a delete request, so it is unnecessary to inform the request sender of that.